src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
-}
+  const dayRentCost = 40;
+  const rebateOfSevenDays = 50;
+  const rebateOfThreeDays = 20;
+  const totalRentalCost = days * dayRentCost;
+
+  if (days >= 7) {
+    return totalRentalCost - rebateOfSevenDays;
+  } else if (days >= 3) {
+    return totalRentalCost - rebateOfThreeDays;
+  }
```

Comment:
There's no need of `else if` statement
```suggestion
  if (days >= 7) {
    return totalRentalCost - rebateOfSevenDays;
  }
  
  if (days >= 3) {
    return totalRentalCost - rebateOfThreeDays;
  }
```